### PR TITLE
Updates instructions for simple wrapper kernel

### PR DESCRIPTION
**Pull request by basheersubei**
migrated from https://github.com/ipython/ipython/pull/8684

### CODE REVIEW

File: docs/wrapperkernels.rst [hunk 1]
```diff
@@ -68,6 +68,8 @@ To launch your kernel, add this at the end of your module::
         from ipyernel.kernelapp import IPKernelApp
         IPKernelApp.launch_instance(kernel_class=MyKernel)
 
+Place your kernel anywhere that you could import it (e.g. current directory). Now create a JSON kernel spec file (see below for example), and place it in ``~/.ipython/kernels/<mykernelname>/``. Now, you can run your kernel using ``ipython console --kernel <mykernelname>``
```
Comment (takluyver) [inline docs/wrapperkernels.rst]: The location to put it in now [depends on the system](http://jupyter.readthedocs.org/en/latest/system.html#data-files). People could use the `jupyter kernelspec install` command to put it in the right place.

Also, `ipython console` -> `jupyter console`

**Comment by basheersubei** [comment]
Is this better now?

**Comment by takluyver** [comment]
Thanks. The mention of `~/.ipython/kernels/<mykernelname>` should be gone, though.

**Comment by basheersubei** [comment]
Oh, so the JSON file should be placed in the same directory as the kernel module file when the kernel is installed (using kernelspec install)?

**Comment by basheersubei** [comment]
fixed again and squash previous commits

**Comment by takluyver** [comment]
Not quite - the kernel module needs to be in a location where Python can import it - the cwd works, so there's no need to copy it anywhere for testing. The kernel spec (the directory containing kernel.json) needs to be in one of those data files locations. You can add a ```:ref:`kernelspecs` ``` to link to [this info](http://jupyter-client.readthedocs.org/en/latest/kernels.html#kernel-specs) about kernel spec.s

**Comment by basheersubei** [comment]
oh, sorry I had no idea there was more documentation. I'll make the changes.

**Comment by takluyver** [comment]
No worries, thanks for your patience on this.

I think it's still a bit confusing. The `jupyter kernelspec` commands are all about the kernel specs - the metadata directories that contain the `kernel.json` file. `jupyter kernelspec install` is to get that in the right place, not to install the module. The `kernelspec` machinery doesn't know about Python modules at all.

So can you move the bit about the `jupyter kernelspec install` command into the previous sentence about kernel specs?

**Comment by basheersubei** [comment]
ah, I see. I misunderstood earlier (should've read the new docs). Is it good now?

**Comment by takluyver** [comment]
Thanks, that's looking good. Can you just add a space before 'Finally', and then I'll merge.

**Comment by basheersubei** [comment]
done

**Comment by takluyver** [comment]
Thanks!

**Pull request merged**